README.md

```diff
@@ -1,3 +1,8 @@
+Front-end redesign
+![screenshot](https://mediabuidlguidl.vercel.app/thumbnail.png)
+
+
+
```

Comment:
remove extra space

Reply:
I think I did this right.. Let me know if you meant something different